docker-gen.go

```diff
@@ -46,11 +46,13 @@ type Event struct {
 }
 
 type Address struct {
-	IP       string
-	Port     string
-	HostPort string
-	Proto    string
-	HostIP   string
+	IP            string
+	IP6_LinkLocal string
```

Comment:
Could you remove the underscores `_`?  I try to keep the code consistent with: https://golang.org/doc/effective_go.html#mixed-caps